src/i18n/pt-br.js

```diff
@@ -0,0 +1,445 @@
+            message: "Existe um id de pagamento associado a este endereço.\nTenha a certeza de copiar este id de pagamanto separadamente."
+        },
+        copyPrivateKeys: {
+            // Copy {seedWords/viewKey/spendKey}
+            title: "Copiar {type}",
+            message: "Tenha cuidado a quem envia as suas chaves privadas, pois elas controlam os seus fundos.",
+            seedWords: "Palavras Semente",
+            viewKey: "Chave de Visualização",
+            spendKey: "Chave de Gasto"
+        },
+        deleteWallet: {
+            title: "Apagar carteira",
+            message: "Tem a certeza absoluta que pretende apagar a sua carteira?\nTenha a certeza que tem um backup da suas chaves privadas.\nESTE PROCESSO NÃO É REVERSÍVEL!",
+            ok: "APAGAR"
+        },
+        exit: {
+            title: "Sair",
+            message: "Tem a certeza que deseja sair?",
+            ok: "SAIR"
+        },
+        keyImages: {
+            title: "{type} imágens chave",
+            message: "Quer {type} imágens chave?",
+            export: "Exportar",
+            import: "Importar"
+        },
+        noPassword: {
+            title: "Nenhum palavra-passe definida",
+            message: "Tem a certeza que deseja criar uma carteira sem palavra-passe?",
+            ok: "SIM"
+        },
+        password: {
+            title: "Palavra-passe",
+            message: "Introduza a sua palavra-passe para continuar"
+        },
+        registerServiceNode: {
+            title: "Registar um nódulo de serviço",
+            message: "Pretende registar um nódulo de serviço?",
+            ok: "REGISTAR"
+        },
+        rescan: {
+            title: "Re-examinar carteira",
+            message: "Aviso: Algumas informações acerca de transações passadas serão perdidas,\ntal como o endereço dos recipientes que serão perdidos.",
+            ok: "RE-EXAMINAR"
+        },
+        restart: {
+            title: "Reiniciar",
+            message: "Alterações requerem um reiniciar. Quer reiniciar agora?",
+            ok: "Reiniciar"
+        },
+        showPrivateKeys: {
+            title: "Mostrar chaves privadas",
+            message: "Deseja ver as suas chaves privadas?",
+            ok: "MOSTRAR"
+        },
+        stake: {
+            title: "Participação num nódulo de serviço",
+            message: "Deseja participar num nódulo de serviço?",
+            ok: "Participação"
+        },
+        sweepAll: {
+            title: "Varrer Tudo",
+            message: "Quer reamente varrer tudo?",
+            ok: "VARRER TUDO"
+        },
+        switchWallet: {
+            title: "Mudar de carteira",
+            closeMessage: "Tem a certeza que deseja fechar a carteira actual?",
+            restartMessage: "A carteira RPC está correntemente em sincronização. \nISe deseja mudar de carteira terá de reiniciar a aplicação. \nPerderá o progresso da sincronização e terá de re-examinar o blockchain."
```

Comment:
restartMessage: "A carteira RPC está em sincronização. \nISe deseja mudar de carteira terá de reiniciar a aplicação. \nPerderá o progresso da sincronização e terá de reescanear o blockchain."

Reply:
"rescanear" is not Portguese. It's a foreign language term with a conjugation. Of course, it can be used as slang, like "download" vs "baixar" - In Brazil "baixar" is extensively used while in Portugal "download" is used. Or "blockchain" vs "cadeia de blocos", the latter is seen once in a while but IMHO makes no sense to translate. In this case, my point is that "rescanear" is very latin america (both pt and es) but is not used in Europe. In European .pt and .es "escanear" directly portrays to a scanner. 

Also, I've used "re-examinar" throughout th

But I've just also realized that under new Acordo Ortográfico it is not even "re-examinar" anymore, as it drops the hifen to the modern "reexaminar" - will review

Reply:
Please refer lower to a more technical exposé of this issue

Reply:
yidakee yeah i completely agree escanear is a non word... but at the same time it is more correct than to use examine. Because it's not examining the blockchain. I felt like escanear was better because it's what bitcoin.org uses in their translation and i think they should be a good reference for consensus on this. https://bitcoin.org/pt_BR/bitcoin-para-pessoas

Honestly some of these term translations are just ridiculous since everyone mentions them by the English term such as node, blockchain, scan, etc. But i believe it is correct to make this translation.